README.md

```diff
@@ -209,7 +209,13 @@ Some native rustc `-Z` flags are also very relevant for Miri:
   functions.  This is needed so that Miri can execute such functions, so Miri
   sets this flag per default.
 * `-Zmir-emit-retag` controls whether `Retag` statements are emitted. Miri
-  enables this per default because it is needed for [Stacked Borrows].
+  enables this per default because it is needed for validation.
+* `-Zmiri-track-pointer-tag=<tag>` shows a backtrace when the given pointer tag
```

Comment:
Something went wrong here, you duplicated the existing help texts instead of expanding it.

Reply:
whoops, looks like I messed up the merge conflict